boxsdk/object/chunked_uploader.py

```diff
@@ -0,0 +1,45 @@
+# pylint: disable=line-too-long
```

Comment:
Also, I think this file should probably be moved out of object/ since it isn't really an API object.  Consider putting it in util/ or something instead